digestbot/core/DBEngine/PostgreSQLEngine.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import asyncpg
+import sys
+
+from digestbot.core.DBEngine.utils.createdb import (
+    create_database,
+    check_exist_tables,
+    create_tables,
+)
+
+
+class PostgreSQLEngine:
+    def __init__(self):
+        self.logger: logging.Logger
+        self.engine: asyncpg.Connection
+
+        self.__setting_logger()
+
+    def __setting_logger(self):
+        """
+        Setting handlers and level of logging for self.logger
+        """
+
+        self.logger = logging.getLogger("DBEngine")
+        self.logger.setLevel(logging.INFO)
+
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
```

Comment:
```suggestion
        handler.setLevel(logging.DEBUG)  # TODO: change after dockerization
```
As stdout is primary log stream in docker I think it's better to either send level info+ messages or (better) make it configurable later